cmd/version/version.go

```diff
@@ -9,7 +9,7 @@ import (
 var (
 	// TODO: This goes away when control-plane-operator becomes another component
 	// in the OCP payload.
-	HyperShiftImage = "registry.ci.openshift.org/hypershift/hypershift:latest"
+	HyperShiftImage = "registry.ci.openshift.org/ocp/4.10:hypershift"
```

Comment:
should this be `registry.ci.openshift.org/ocp/latest:hypershift` or so?

Reply:
We can not change this, because the image here must be public, otherwise `hypershift install` will not result in a working hypershift setup anymore.

Reply:
Not sure, @alvaroaleman @csrwng thoughts?

Reply:
If we leave it at `latest` then when we cut a new branch wouldn't the branch track the wrong version (e.g. `latest`)?

Reply:
>If we leave it at latest then when we cut a new branch wouldn't the branch track the wrong version (e.g. latest)?

I think not necessarily. E.g we could say the CLI release cycle is independent from HyperShift itself and it should always point to latest operator unless explicitly passing otherwise. Or we could have release automation that sets the pinned image as part of release cut process. I guess we need to discuss and define the release process, until then probably keep the scope of the changes the smallest possible.